scapy/sendrecv.py

```diff
@@ -1321,6 +1399,10 @@ def join(self, *args, **kwargs):
             self.thread.join(*args, **kwargs)
 
 
+AsyncSniffer.__doc__ = ((AsyncSniffer.__doc__ or "") + _DOC_SNIFF_PARAMS +
```

Comment:
Why the or?

Reply:
AsyncSniffer.__doc__ can be `None | str`, therefore the + operation is not defined

Reply:
I'd prefer to `# type: ignore` in that case

Reply:
Done